xapian-applications/omega/omindexcheck.cc

```diff
@@ -133,7 +133,7 @@ index_test()
 #endif
 #if defined HAVE_LIBABW
     tests.insert({"abw/test.abw",
-		  {"ZAparth", "ZSabiword", "Zsampl", "Zdocument"}});
+		  {"ZAparth", "Zabiword", "Zsampl", "Zdocument"}});
```

Comment:
I said on IRC adding a comment here would be good:

> [07:58:00] <olly> so probably use something other than the document title in this testcase
> [07:58:26] <olly> and note in a comment that we deliberately aren't testing the title, because of this bug fix not being present in some versions still in use

It's useful to note in the code when there's a reason why we don't test something that it might seem to someone reading the code that it would be good to test (i.e. a term from the subject here), since otherwise they might try to be helpful and improve the test coverage by adding `ZSabiword` or similar to the terms checked, and because developer machines tend to have fairly recent versions of everything it'll probably work fine for them and then we'll have to do another loop around the cycle of discovering it doesn't work with older libabw versions and taking that back out again.

If you also note the version of libabw this was fixed in (which you can see from the libabw commit I linked to) then the comment also serves to tell us what version of libabw we need to assume if we want to restore this check in the future.

Reply:
Thanks for pointing this out. I have fixed this.